server/server.js

```diff
@@ -94,6 +109,21 @@ app.post(`${apiBaseUrl}/participants`,
 
     try {
       const response = await parseAndSaveParticipants(req.file.buffer);
+
+      // HCAP-244 - Log Participant Upload
+      const { content } = req.kauth.grant.access_token;
+      const logData = {
+        "action": "participant_post",
+        "uploaded_by": {
```

Comment:
I think I would try to harmonize the format of these logs a little bit - like I think that instead of "uploaded_by" and "accessed_by" we can change both to "performed_by" since we have a separate key for the action already. Similar for "ids_viewed" and "ids_posted" - we could probably change both of these to "participant_ids"